webapp/src/components/properties/link/link.tsx

```diff
@@ -15,10 +15,13 @@ type Props = {
     onSave: () => void
     onCancel: () => void
     validator: (newValue: string) => boolean
+    tooltip?: string
 }
 
 const URLProperty = (props: Props): JSX.Element => {
     let link: ReactNode = null
+    const tooltipClassName = props.tooltip ? 'octo-tooltip tooltip-top' : ''
```

Comment:
This code would repeat over and over in any new property component that gets added in the future. Also since the logic for adding and generation tooltip is independent of property type, I'd like to remove it from each individual component.

Instead, I think we can add a wrapper `div` with tooltip over here - 

https://github.com/mattermost/focalboard/blob/6cd5003a7aa67ff11fb1bcaca4a46914a8730428/webapp/src/components/kanban/kanbanCard.tsx#L78

This is where the visible properties are added to the kanban card component. That part of the code is only used for readonly view so we can skip the logic of deciding whether to add the tooltip or not. This will have the following advantages -

1. No repeating code in each property component (there are a bunch of new ones being added in the open PRs)
2. Tooltips for new types gets added automatically without any code change.

This approach can also be applied to the gallery view which suffers from the same problem.

Reply:
Thanks @harshilsharma63, this is a better way of handling it, I'll will update it.

Reply:
Hi @harshilsharma63,

With regards to adding the same functionality to the gallery view, should one create a new ticket for it so there is a track record of the change or should I add it while I am busy?

Currently I only updated the current Kanban card.

Reply:
If it's not too much extra work, I think getting gallery view done within this same PR would be great unless you are busy and don't have the time to do it which we can create a new PR for it. Thoughts @harshilsharma63? I'll make it your call but I ultimately think it makes sense to do it one PR.

Reply:
@ditsemto sorry I missed your reply. If it's a quick thing then we can add it for gallery view as well. If not, we can handle it separately.

Reply:
@harshilsharma63 No worries. 

I made the requested changes on one commit and updated the gallery view in the second commit.